js/rebase-client/src/lib.rs

```diff
@@ -33,40 +34,56 @@ pub struct Client {
     client: Arc<RebaseClient>,
 }
 
+#[wasm_bindgen]
+#[derive(Deserialize, Serialize)]
+pub struct Config {
+    instructions: String,
+    statement: String,
+    jwt: Option<String>,
+    ld: Option<String>,
+    verify_jwt: Option<String>,
+    verify_ld: Option<String>,
```

Comment:
if you want to avoid the `URL::parse` calls in `::new`, you can use the `serde_with` crate with `#[serde_as(as = "DisplayFromStr)]` to deserialise URL directly. the current implementation is of course fine though, I'm just a `serde_with` fan

Reply:
That's a good idea, I'll take a look at doing that in the next PR (#60) where I'm deeply in the rebase-client code.

Reply:
I realised that the `url` crate has a `serde` feature for this, so no need for `serde_with`